src/index.js

```diff
@@ -1,16 +1,34 @@
 import React from 'react';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
 import ReactDOM from 'react-dom';
-import App from './App';
+import App from './components/App';
+import bookReducer from './reducers/books';
 import reportWebVitals from './reportWebVitals';
 
+const initialState = {
+  books: [
+    {
+      title: 'Learn React',
+      category: 'Learning',
+      id: Math.floor(Math.random() * 1000),
+    },
+    {
+      title: 'Learn Redux',
+      category: 'Learning',
+      id: Math.floor(Math.random() * 1000),
+    },
```

Comment:
### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'Learn React', category: 'Learning'}
```